assets/app/view/game/corporation.rb

```diff
@@ -45,53 +37,46 @@ def render
 
         children << render_companies if @corporation.companies.any?
 
-        revenue_table_props = {
-          style: {
-            'text-align': 'center',
-            'font-weight': 'bold',
-            'margin-left': 'auto',
-            'margin-right': 'auto',
-          },
-        }
-
         if @corporation.owner
           subchildren = (@corporation.operating_history.empty? ? [] : [render_revenue_history])
-          children << h(:table, revenue_table_props, [h(:tr, subchildren)])
+          children << h('table.revenue', [h(:tr, subchildren)])
```

Comment:
remove all of the CSS inheritance, make a method that does inline styles for the table components that you want